git-keeper-client/gkeepclient/server_actions.py

```diff
@@ -423,7 +423,9 @@ def disable_assignment(class_name: str, assignment_name: str,
     print('Disabling assignment {} in class {}. This cannot be undone.'
           .format(assignment_name, class_name))
     print('Students will be notified that the assignment has been disabled.')
-    print('Tests will no longer be run on submissions to this assignment.')
+    print('The action.sh/action.py script will no longer be run on submissions.')
```

Comment:
The term "action script" covers both action.sh and action.py, and makes it easier if we add support for another action script in the future. There is also a period in the middle of the sentence that continues on the next line.